runtime/colors/blue.vim

```diff
@@ -13,12 +13,12 @@ set background=dark
 hi clear
 let g:colors_name = 'blue'
 
-let s:t_Co = exists('&t_Co') && !empty(&t_Co) && &t_Co >= 0 ? &t_Co : -1
+let s:t_Co = exists('&t_Co') ? (&t_Co ? &t_Co : 0) : -1
```

Comment:
Maybe comment out the original line rather than delete (it may be ported in the future)

Reply:
Really? I thought we locked vimscript at v1.

Reply:
it's available in v1

Reply:
Oh. Still, it's easy enough to update manually (and no big loss if it is forgotten)...